README.md

```diff
@@ -1 +1,49 @@
-# Portfolio
\ No newline at end of file
+# Project Name
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
```

Comment:
- [ ] This README is well structured and looks professional. However, it does not reflect any thing about the project you are working on. I think it would be nice if you customized it to reflect information about your project 😄.

Reply:
I have modified the project as requested. I am trying to submit another code review request but it doesn't give me the option on dashboard. Can you please, let me know what should be done?